src/Particular.CodeRules/tools/install.ps1

```diff
@@ -0,0 +1,6 @@
+﻿param($installPath, $toolsPath, $package, $project)
```

Comment:
So the install script copies it to the correct place on install. But if the binarie is not checked in to source control what puts it in the correct location on package restore?